setup/scaling/caching/configure-redis-usage.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: single
+title:  "Configure Redis Usage"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker relies on its Redis cache for a number of reasons: caching
+infrastructure, storing live executions, returning pipeline definitions faster,
+etc... Each of these can be configured to your needs, whether it is to make
+Spinnaker more responsive, or reduce the load on a downstream dependency.
+
+## Configure infrastructure caching
+
+By default, Spinnaker's Clouddriver service will spawn "caching agents" every
+30 seconds. These agents are scheduled and coordinated across all instances of
+Clouddriver, allowing you to reduce the load on any single instance of
+Clouddriver by creating more replicas of the service.
+
+To adjust how this caching happens, Clouddriver exposes a few properties that
+can be set in `~/.hal/$DEPLOYMENT/profiles/clouddriver-local.yml`:
+
+```yaml
+# How many seconds (default 30) between runs of agent. Lowering this number
+# means the resources in the Spinnaker UI will be updated more frequently,
+# at the cost higher API/quota usage of your cloud provider.
```

Comment:
So, to optimize for scale use a higher number?

Reply:
Possibly -- unless you need fresher data.